src/test/test_type_domain.cpp

```
@@ -0,0 +1,565 @@
+// Unit tests for the DSU-based TypeDomain.
+
+#include <catch2/catch_all.hpp>
+
+#include "arith/dsl_syntax.hpp"
+#include "crab/type_domain.hpp"
+#include "crab/type_encoding.hpp"
+#include "crab/var_registry.hpp"
+
+using namespace prevail;
+
+static constexpr Reg r0{0};
```

Clang (14.0.6)

[warning] variable name 'r0' is too short, expected at least 3 characters

(readability-identifier-length)